src/importexport/musicxml/tests/tst_mxml_io.cpp

```diff
@@ -262,6 +264,17 @@ static void fixupScore(Score* score)
     score->setSaved(false);
 }
 
+static void checkScore(Score* score)
```

Comment:
Maybe have comment atop that explains the purpose of this new method?

Reply:
You mean like the one that explains the purpose of fixupScore?  In that case it would read:
"checkScore -- do required checks after MusicXML import"

I suppose at least it has the advantage of consistency...

Reply:
yes

Reply:
That's the sort of comment I'd ask developers to remove in the teams I normally work in!

Reply:
Be more explicit then:
"Check whether scores/parts with more than one staff have the final barline connected"